config/metricConfigs/learnMetrics.ts

```diff
@@ -389,6 +389,13 @@ const allowedMetrics: MetricsConfig = [
     labels: [],
     protocol: "statsd"
   },
+  {
+    name: "campus__time_to_submit_code",
+    help: "measures the time to submit a code and get the response",
+    type: "distribution",
+    labels: [],
```

Comment:
The labels here don't seem to do anything, so skipped these for now. Feel free to disagree.

Reply:
I agree that they don't seem to do anything other than document the intended fields